pkgs/by-name/pd/pdm/package.nix

```diff
@@ -74,7 +77,8 @@ python.pkgs.buildPythonApplication rec {
       id
       virtualenv
     ]
-    ++ httpx.optional-dependencies.socks;
+    ++ httpx.optional-dependencies.socks
+    ++ optional-dependencies.msgpack;
```

Comment:
`optional-dependencies` should not be added to `dependencies`.
If needed for tests, please add it to `nativeCheckInputs` instead.